src/NServiceBus.Core/Installation/InstallationComponent.cs

```diff
@@ -76,8 +76,6 @@ public class Configuration
             public Configuration(SettingsHolder settings)
             {
                 this.settings = settings;
-
-                settings.SetDefault("Transport.CreateQueues", true);
```

Comment:
I guess that this could technically break a downstream that relies on this being there?

Search comes up blank tho

Reply:
why? the value is now set here: https://github.com/Particular/NServiceBus/pull/5525/files#diff-bb50e55af39469cabb99ce666e7b5d97R368 it's just not a default value anymore but a regular value

Reply:
Yes but a downstream doing settings.Get<bool>("Transport.CreateQueues") will blow up right?

Reply:
Ok, I see it now